.github/workflows/ci.yml

```diff
@@ -93,4 +93,25 @@ jobs:
             rm diff.txt || true # we don't fail if the file does not exist
           done;
           exit $code
+  make-httpd-maintainer-mode:
+    runs-on: ubuntu-latest
+    name: "Build with httpd in maintainer-mode"
```

Comment:
If we are putting human readable names to jobs, then lets do it for all or none.

Reply:
Good point, removed.